src/java/org/sakaiproject/feedback/tool/entityproviders/FeedbackEntityProvider.java

```diff
@@ -136,6 +140,7 @@ private String handleReport(final EntityView view, final Map<String, Object> par
         final String screenSize = screenWidth + " x " + screenHeight + " pixels";
         final String plugins = (String) params.get("plugins");
         final String ip = requestGetter.getRequest().getRemoteAddr();
+        final String currentTime = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", new ResourceLoader("org.sakaiproject.feedback").getLocale()).format(new Date());
```

Comment:
Won't this still use the date format you supplied rather than a locale appropriate one?